app/templates/main/volunteerProfile.html

```diff
@@ -23,9 +23,11 @@ <h6 align="center">{{volunteer.phoneNumber}}</h6>
      <li class="nav-item" role="presentation">
        <a class="nav-link active" aria-current="page" href="#">Upcoming Events</a>
      </li>
+     {% if g.current_user.isCeltsAdmin or g.current_user.username in volunteer.username %}
```

Comment:
We should use `==` instead of `in`, to make sure the user is actually the same user